### DISCUSSION TITLE
24-02-23 불참 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [ThinkAboutSoftware/AcademicConference#382] [General]
오늘 개인 스케줄로 인해서 참여가 어려울 것 같습니다. 잠깐 참여라도 가능하다면 참여 해볼 수 있는데, 힘들 것 같아서 미리 말씀드립니다.

**Comment by jongfeel** [comment]
알겠습니다.

미리 내용 정리하신 걸 올려 주셔서 나중에 다른 분들 의견에 코멘트 해 주시고 리뷰 해 주시면 좋겠습니다.
